hs_core/tasks.py

```diff
@@ -150,8 +150,7 @@ def nightly_zips_cleanup():
     istorage = IrodsStorage()
     if istorage.exists(zips_daily_date):
         istorage.delete(zips_daily_date)
-    federated_prefixes = BaseResource.objects.all().values_list('resource_federation_path')\
-        .distinct()
+    federated_prefixes = BaseResource.objects.order_by().values_list('resource_federation_path').distinct()
```

Comment:
This change removes retrieving duplicate `resource_federation_path` values which is nice. However, I am not sure why it causes recursion error when we have duplicate values.